### ISSUE TITLE
Can't create VMAgent through chart victoria-metrics-operator with RBAC missing error

### ISSUE THREAD

**Issue by linjunzhe** [VictoriaMetrics/helm-charts#708]
I tried to create vmagent, but got following logs in vmoperator. 
```
{"level":"error","ts":1696568468.2472074,"msg":"Reconciler error","controller":"vmagent","controllerGroup":"operator.victoriametrics.com","controllerKind":"VMAgent","VMAgent":{"name":"agent","namespace":"monitoring"},"namespace":"monitoring","name":"agent","reconcileID":"dd55619f-dcee-4b9d-8e72-83fd07e24a66","error":"cannot create vmagent role and binding for it, err: cannot ensure state of vmagent's cluster role: clusterroles.rbac.authorization.k8s.io \"monitoring:vmagent-cluster-access-agent\" is forbidden: user \"system:serviceaccount:monitoring:vmoperator-victoria-metrics-operator\" (groups=[\"system:serviceaccounts\" \"system:serviceaccounts:monitoring\" \"system:authenticated\"]) is attempting to grant RBAC permissions not currently held:\n{APIGroups:[\"discovery.k8s.io\"], Resources:[\"endpointslices\"], Verbs:[\"get\" \"list\" \"watch\"]}","stacktrace":"sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler\n\tsigs.k8s.io/controller-runtime@v0.13.1/pkg/internal/controller/controller.go:326\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem\n\tsigs.k8s.io/controller-runtime@v0.13.1/pkg/internal/controller/controller.go:273\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func2.2\n\tsigs.k8s.io/controller-runtime@v0.13.1/pkg/internal/controller/controller.go:234"}
```
The SA has no permission to get/list/watch endpointslices. Seems it's missing in ClusterRole. 
```
# kubectl -n monitoring auth can-i list endpointslices --as=system:serviceaccount:monitoring:vmoperator-victoria-metrics-operator
no
```
I think it's related to the recent commit https://github.com/VictoriaMetrics/operator/commit/958ce2b829436d0c1e00f1c173964024c6890b7f in vmoperator. The chart needs to be modified accordingly to fit this change. 

### Version info
Chart victoria-metrics-operator:0.27.2
VM Operator 0.39.0
VMAgent  1.93.5

**Comment by romancin** [comment]
Same problem here.

**Comment by SammyA** [comment]
Me too, rolled back to chart v0.27.1 .

**Comment by jicki** [comment]
Same problem here.

**Comment by Haleygo** [comment]
Should be addressed in https://github.com/VictoriaMetrics/helm-charts/pull/713 and released in [victoria-metrics-operator-0.27.3](https://github.com/VictoriaMetrics/helm-charts/releases/tag/victoria-metrics-operator-0.27.3) & [victoria-metrics-k8s-stack-0.18.5](https://github.com/VictoriaMetrics/helm-charts/releases/tag/victoria-metrics-k8s-stack-0.18.5).
Close as completed, feel free to reopen if it's not.
